Kajabi/sage-lib
# [Sage 814] Banner icon updates

Resolves #814 

## Description

- `icon` as a property type of Banner component is not valid because designs define clear usage of banner icons. This PR removes the code example that indicates `icon` is a valid property
- Update the Announcement icon to align with [designs](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/%5BWIP%5D-Sage-3-%E2%80%94-Admin-Components?node-id=6999%3A21606)
    - FYI: [Design confirms](https://kajabi.slack.com/archives/C01A424HY8Y/p1632321794026700) that icon is supposed to be a flag

## Screenshots
|  Before code example  |  After code example  |
|--------|--------|
|<img width="941" alt="Screen Shot 2021-09-22 at 10 29 06 AM" src="https://user-images.githubusercontent.com/24237393/134373975-3a8b9509-b5cd-4b26-9bd1-a66136b2fae8.png">|<img width="951" alt="Screen Shot 2021-09-22 at 10 30 34 AM" src="https://user-images.githubusercontent.com/24237393/134374185-1e18f296-df1a-46ef-8170-1b2bcf83178a.png">|

|  Before icon change  |  After icon change |
|--------|--------|
|<img width="651" alt="before-banner-icon" src="https://user-images.githubusercontent.com/24237393/134371551-00020e85-5d5f-4b3c-9fcc-6cb2baa58a0d.png">|![after-banner-icon](https://user-images.githubusercontent.com/24237393/134371547-3883c616-50ae-465c-9c24-9aea3fe7276f.jpg)|

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/banner
2. Observe the Code examples no longer include `icon:` as a property
3. Also observe the Announcement icon now aligns with [designs](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/%5BWIP%5D-Sage-3-%E2%80%94-Admin-Components?node-id=6999%3A21606), using a flag rather than a megaphone

## Testing in `kajabi-products`
1. (**LOW**) Documentation change for icon usage in Banner; no impact on `kajabi-products`
